lib/kubernixos.nix

```diff
@@ -2,36 +2,40 @@
 let
   pkgs = if builtins.isAttrs packages then packages else (import packages {});
 
+  cfg = (import "${toString pkgs.path}/nixos/lib/eval-config.nix" {
+    inherit pkgs modules;
+  }).config.kubernixos;
+
+  kubernixos = with builtins; substring 0 32 (hashString "sha256" (toJSON cfg.manifests));
```

Comment:
Why are we only using the first 32 characters of the hash?

Reply:
which.. actually is 33 chars now (from 0-32) ;-) I'm open for suggestions here? Should we go for absolute maximum (63 chars) or rather leave some room for encoding other information in there in the future?